### PR TITLE
10 build after updating

### PR DESCRIPTION
**Pull request by andyzickler**
Ref #10 

Add missing build step to circleci config

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -54,6 +54,9 @@ jobs:
             git config user.email "support@spry-group.com"
             git config user.name "Spry Bot"
             git commit -m $'chore: update content.json\n\n[skip ci]'
```
Comment (dopry) [inline .circleci/config.yml]: We only need one commit here when everything succeeds.

**Review by dopry — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (andyzickler) [reply .circleci/config.yml]: @dopry updated

File: .circleci/config.yml [hunk 2]
```diff
@@ -53,6 +53,8 @@ jobs:
             git add src/201901-CDP/content.json
             git config user.email "support@spry-group.com"
             git config user.name "Spry Bot"
+            npm run build
```
Comment (dopry) [inline .circleci/config.yml:56]: I think you need to run both npm update-data and npm build.

**Review by dopry — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (andyzickler) [reply .circleci/config.yml:56]: That already happens in a previous step

**Review by dopry — APPROVED** [review]

**Pull request merged**